charts/karpenter/templates/clusterrole.yaml

```
@@ -37,5 +37,24 @@ rules:
     resources: ["daemonsets"]
     verbs: ["list", "watch"]
   - apiGroups: ["admissionregistration.k8s.io"]
-    resources: ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
-    verbs: ["get", "watch", "list", "update"]
+    resources: ["mutatingwebhookconfigurations"]
+    verbs: ["get", "watch", "list", "update", "delete"]
+    resourceNames: ["defaulting.webhook.provisioners.karpenter.sh"]
+  - apiGroups: ["admissionregistration.k8s.io"]
+    resources: ["validatingwebhookconfigurations"]
+    verbs: ["get", "watch", "list", "update", "delete"]
```

Need to break out the `list` and `watch` so that they are not constrained by a specific resourceName

```suggestion
  - apiGroups: ["admissionregistration.k8s.io"]
    resources: ["mutatingwebhookconfigurations", "validatingwebhookconfigurations"]
    verbs: ["list", "watch"]
  - apiGroups: ["admissionregistration.k8s.io"]
    resources: ["mutatingwebhookconfigurations"]
    verbs: ["get", "update", "delete"]
    resourceNames: ["defaulting.webhook.provisioners.karpenter.sh"]
  - apiGroups: ["admissionregistration.k8s.io"]
    resources: ["validatingwebhookconfigurations"]
    verbs: ["get", "update", "delete"]
```